core/backend/src/IpcHost.ts

```diff
@@ -214,16 +215,15 @@ class IpcAppHandler extends IpcHandler implements IpcAppFunctions {
     return IModelDb.findByKey(key).nativeDb.getUndoString();
   }
 
-  public async pullAndMergeChanges(key: string, version?: IModelVersionProps): Promise<string> {
+  public async pullAndMergeChanges(key: string, version?: IModelVersionProps): Promise<ChangesetIndexAndId> {

```

Comment:
@kabentley the change in return type here may be worth a mention in the NextVersion.md so people know how to respond to the change.